content/modules/mobile-travel-tutorial/partials/installation/manual.adoc

```diff
@@ -176,22 +170,30 @@ To start the service, you can use the Services application (Control Panel --> Ad
 `{"couchdb":"Welcome","vendor":{"name":"Couchbase Sync Gateway","version":"3.0"},"version":"Couchbase Sync Gateway/3.0.0(460;26daced) EE"}`
 ****
 
-
 == Python Travel Sample Web Backend
 
+=== Apple M1 Users:
 
-=== Windows Users
+* Clone the `mobile-travel-sample-m1` branch of Travel Sample Python web app repo
++
+[source,bash]
+----
+git clone -b mobile-travel-sample-m1  https://github.com/couchbaselabs/try-cb-python.git
+cd try-cb-python
+----
 
-* The instructions in this section are for Windows users.
-Scroll to "Other Platforms" section for instructions for other platforms.
+==== Other platforms:
 
 * Clone the `mobile-travel-sample-tutorial` branch of Travel Sample Python web app repo
 +
 [source,bash]
 ----
-$ git clone -b mobile-travel-sample-tutorial  https://github.com/couchbaselabs/try-cb-python.git
+git clone -b mobile-travel-sample-tutorial  https://github.com/couchbaselabs/try-cb-python.git
+cd try-cb-python
 ----
 
+*Windows Users Only*
```

Comment:
Move "Windows Users Only" and "Others" to an "installation guide" section - otherwise it seems like these instructions are related to "Other Platforms"
Also "View File" to see how it looks